ui/components/MesherySettingsPerformanceComponent.js

```diff
@@ -285,8 +287,260 @@ class MesherySettingsPerformanceComponent extends React.Component {
       </NoSsr>
     );
   }
-}
+}*/
+
+function MesherySettingsPerformanceComponent(props) {
+  const { classes, qps: initialQPS, c: initialC, t: initialT, gen: initialGen, notify } = props;
```

Comment:
I think these names `c` and `t` aren't very descriptive can we make them more descriptive?

Reply:
yes sir we can make names descriptive.

Reply:
Thankyou

Reply:
sir, please check I have updated the correction. If there is anything wrong please let me know and guide me.

Reply:
sir, please check I have updated the correction. If there is anything wrong please let me know and guide me.